.github/scripts/export-keycloak-version.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+if [ ! -d "keycloak-dist" ]; then
+  exit 0
+fi
+
+if [ -n "$GITHUB_BASE_REF" ] && [[ "$GITHUB_BASE_REF" == "latest" ]]; then
+  export KEYCLOAK_VERSION=$(grep -oPm1 "(?<=<version>)[^<]+" pom.xml)
```

Comment:
I think we could use a similar command as below:
```suggestion
  export KEYCLOAK_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
```

Reply:
Done